commons-numbers-complex-arrays/src/main/java/org/apache/commons/numbers/complex/arrays/ComplexList.java

```diff
@@ -138,48 +139,122 @@ private void rangeCheckForInsert(int index) {
 
     /**
      * Gets the complex number \( (a + i b) \) at the indexed position of the list.
-     *
+     * <p>
      * {@inheritDoc}
+     *
      * @return the complex number.
      */
     @Override
     public Complex get(int index) {
         rangeCheck(index);
-        final int i = index << 1;
```

Comment:
The precomputed value i will be reused so it can be left as before, please revert.